### DISCUSSION TITLE
如何获取表格组件的行信息

### DISCUSSION THREAD

**Discussion by ChrisLo0751** [opentiny/tiny-engine#209] [General]
需求：当点击表格组件的编辑项时，可以获取该行的信息

如图所示：

![图片](https://github.com/opentiny/tiny-engine/assets/73991742/0d5d68d6-8e0c-4f71-9b36-3931c8cbe516)

我希望点击编辑时，获取该行的信息

参考这个页面： https://opentiny.design/tiny-vue/zh-CN/os-theme/components/grid-operation-column

![图片](https://github.com/opentiny/tiny-engine/assets/73991742/3fb50bb0-461e-41d2-aca8-e65c27905ffe)

**Comment by ChrisLo0751** [comment]
@chilingling

**Reply by chilingling** [reply]
@ChrisLo0751 
1. 列数据开启插槽、默认插槽内拖入 button（或者是自定义的组件）
![image](https://github.com/opentiny/tiny-engine/assets/26962197/5ee86534-2466-4cd3-979d-42b428635e73)

2. 给按钮绑定点击事件

![image](https://github.com/opentiny/tiny-engine/assets/26962197/1f92a4d9-e638-4c19-8018-52a03ca314f1)

3. 点击事件内获取到的第二个参数就是行数据
![image](https://github.com/opentiny/tiny-engine/assets/26962197/96a52107-d3ad-4347-bf27-8ec677bc538c)

4. 查看实际出码
![image](https://github.com/opentiny/tiny-engine/assets/26962197/24e9c49e-d6a2-4709-8489-ec76f68ba5d6)
